onnxruntime/core/providers/cpu/ml/svmclassifier.cc

```diff
@@ -75,6 +75,31 @@ SVMClassifier<T>::SVMClassifier(const OpKernelInfo& info)
   }
 }
 
+template <typename LabelType>
+int _set_score_svm(Tensor* Y, const float& maxweight, const int64_t& maxclass, const int64_t& n,
+                   POST_EVAL_TRANSFORM post_transform_, const std::vector<float>& proba_, bool weights_are_all_positive_,
+                   const std::vector<LabelType>& classlabels, const LabelType& posclass, const LabelType& negclass) {
+  int write_additional_scores = -1;
+  if (classlabels.size() == 2) {
+    write_additional_scores = post_transform_ == POST_EVAL_TRANSFORM::NONE ? 2 : 0;
+    if (proba_.size() == 0) {
+      if (weights_are_all_positive_ && maxweight >= 0.5)
+        Y->template MutableData<LabelType>()[n] = classlabels[1];
```

Comment:
>Y->template MutableData<LabelType>() [](start = 8, length = 36)

Create a local var and call it output_data

Reply:
done